content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -5,24 +5,24 @@ kind: tutorial
+Kata on Codewars are created by regular users who want to share their ideas with the community and create challenges for others to train on. After solving a few kata, many users often find that they want to contribute back to the community with kata of their own. Not only can authoring a kata be a great learning experience, well-crafted kata are a source of great fun for everyone.
 
-However, authors sometimes do not realize the fact that creating a good quality kata is much harder than solving it. Creating kata is a totally different kind of task from solving kata. It requires a much wider mindset and set of skills. Creating a kata is much closer to "professional" tasks: it requires you to design a task with a large set of target users in mind, write a solution, create tests for it, handle feedback from users, and maintain it by fixing bugs. Every kata is a small software product in itself, and goes through a small equivalent of a full software development lifecycle!
+However, first-time authors – regardless of programming experience – may not realize that creating a good quality kata from scratch is a much more involved task than solving one, as it requires a wider mindset and broader set of skills (if that sounds intimidating, don't be discouraged: that's what this tutorial is for!). In many ways authoring a kata is closer to what would be considered a "professional" task: it requires designing a challenge with a diverse target audience in mind, writing a solution, creating tests, handling feedback from users, and finally maintaining it via bug-fixes. Every kata is a like a small software product in and of itself and goes through the equivalent of a full software development lifecycle.
 
 ## Support for authors
 
-To support you with this difficult task, a set of help pages has been created with the following types of information:
+To support you in this challenging task, we have provided some helpful resources:
 
-- Tutorials for users who are still figuring things out.
-- [Guidelines](/authoring/guidelines/), which need to be respected to meet the Codewars quality criteria. It is strongly recommended to become familiar with these, otherwise you risk that your kata will be met with bad reception, harsh remarks, and many reported issues.
-- Recipes explaining how to realize some commonly occurring tasks, or solve commonly repeating problems.
-- [Language specific pages](/languages/) with code snippets, examples, references, and tutorials related to specific programming languages.
+- Tutorials for users new to the authoring process.
+- [Guidelines](/authoring/guidelines/) which must be respected in order to meet the Codewars quality criteria. It is strongly recommended to become familiar with these, otherwise you risk receiving negative feedback on your newly created kata, including numerous issue reports and possibly harsh remarks.
```

Comment:
```suggestion
- [Guidelines](/authoring/guidelines/) which must be respected in order to meet the Codewars quality criteria. It is strongly recommended becoming familiar with these, otherwise you risk receiving negative feedback on your newly created kata, including numerous issue reports and possibly harsh remarks.
```